triage/helpers.py

```diff
@@ -146,15 +147,33 @@ def format_csv_rows(cls):
         return {'HS' + row['commodity_code']: row for row in csv_rows}
 
 
+class CountryExportTotalsCSVComtradeFile(BaseCSVComtradeFile):
+    file_path = 'triage/resources/country_commodity_export_totals.csv'
+
+    @classmethod
+    def format_csv_rows(cls):
+        csv_rows = cls.load_csv_rows()
+        formatted = defaultdict(dict)
+        for row in csv_rows:
+            formatted['HS' + row['commodity_code']][row['partner_iso']] = (
```

Comment:
This is not really readable, too many square brackets. Can you please break it down?